app/models/ext_management_system.rb

```diff
@@ -36,6 +36,10 @@ def self.supported_types_and_descriptions_hash
     supported_subclasses.each_with_object({}) { |klass, hash| hash[klass.ems_type] = klass.description }
   end
 
+  def self.subclasses_support?(feature)
```

Comment:
```suggestion
  def self.subclasses_supports?(feature)
```

or perhaps `subclasses_supporting?`?  I think I like `subclasses_supports?` the best only because it it mirrors the method we are calling (i.e. `subclasses.supports?`)